server/market/market.go

```diff
@@ -190,6 +253,44 @@ func (m *Market) waitForEpochOpen() {
 	<-c
 }
 
+// Status describes the operation state of the Market.
+type Status struct {
+	Running       bool
+	EpochDuration uint64 // to compute times from epoch inds
+	ActiveEpoch   int64
+	StartEpoch    int64
+	SuspendEpoch  int64
+	PersistBook   bool
+}
+
+// Status returns the current operating state of the Market.
+func (m *Market) Status() *Status {
+	m.epochMtx.Lock()
+	defer m.epochMtx.Unlock()
+	return &Status{
+		Running:       m.Running(),
+		EpochDuration: m.marketInfo.EpochDuration,
+		ActiveEpoch:   m.activeEpochIdx,
+		StartEpoch:    m.startEpochIdx,
+		SuspendEpoch:  m.suspendEpochIdx,
+		PersistBook:   m.persistBook,
+	}
+}
+
+// Running indicates is the market is accepting new orders. This will return
+// false when suspended, but false does not necessarily mean Run has stopped
+// since a start epoch may be set.
```

Comment:
Should document that it's of limited use and communicating subsystems shouldn't rely on the result since a market could start or stop.  Other subsystems should know or be informed of market running/suspend status rather than rely on this.  e.g. The use of `Running` in `OrderRouter` is an better handled by the `OrderRouter` itself tracking market statuses and handling incoming orders accordingly, however the check is OK for now since the Market itself handles orders that do sneak in while the market is in the process of shutting down.